### ISSUE TITLE
Issue with Wifi and Audio in new version.

### ISSUE THREAD

**Issue by pswiifanwmi** [maxnet/berryboot#330]
I was using the Berryboot from March and I decided to upgrade to the newer one.  I replaced all of the files with ones in the .zip folder except for the config, cmdline, and uenv files.  Now when I test it with both my Pi 2 and Pi 3, wifi and audio don't work when they did in the old version.  Is this a kernel issue or did I do something wrong?

**Comment by maxnet** [comment]
With the newest Pi2/Pi3 version you can just replace all files and select "use existing files" in the disk selection screen.

If you want to replace files manually you'll need to extract shared.tgz to the /shared folder on ext4 partition.

**Comment by pswiifanwmi** [comment]
Yeah, I just figured that out.  Thanks anyway, and WOW!  That was a smart thing to implement!
